dyno-jedis/src/test/java/com/netflix/dyno/jedis/utils/SSLContextUtil.java

```diff
@@ -0,0 +1,36 @@
+package com.netflix.dyno.jedis.utils;
+
+import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManagerFactory;
+import java.io.InputStream;
+import java.security.KeyStore;
+import java.security.SecureRandom;
+
+public class SSLContextUtil {
+
+    /**
+     * hardcoded password for both keystore/truststore for client and server. Because of we are using
+     * selfsigned certificates generated only for this purpose, it is perfectly ok to have publicly aviable password here.
+     */
+    private static final String STOREPASS = "dynotests";
+
+    public static final SSLContext createAndInitSSLContext(final String jksFileName) throws Exception {
+        final ClassLoader classloader = Thread.currentThread().getContextClassLoader();
+        final InputStream inputStream = classloader.getResourceAsStream(jksFileName);
+
+        final KeyStore trustStore = KeyStore.getInstance("jks");
+        trustStore.load(inputStream, STOREPASS.toCharArray());
+
+        final KeyManagerFactory keyManagerFactory = KeyManagerFactory.getInstance(KeyManagerFactory.getDefaultAlgorithm());
+        keyManagerFactory.init(trustStore, STOREPASS.toCharArray());
+
+        final TrustManagerFactory trustManagerFactory = TrustManagerFactory.getInstance(TrustManagerFactory.getDefaultAlgorithm());
+        trustManagerFactory.init(trustStore);
+
+        final SSLContext sslContext = SSLContext.getInstance("TLS");
```

Comment:
While setting up the server & client not a part of this library or PR, it would nice to set the integration test server and client up to only use TLS 1.2.

Reply:
can we treat JedisConnectionFactoryIntegrationTest.java as integration test ? there is full ssl connection established.

Reply:
@KowalczykBartek my understanding is that @russell-lewis is more worried about the TLS version. Would this improve the TLS version `SSLContext sslContext = SSLContext.getInstance("TLSv1.2");`?

Reply:
Updated.

P.S 
even is "TLS" is used, server will respond with highest possible for client and server version.